spec/api-browser-window-spec.js

```diff
@@ -1191,6 +1191,41 @@ describe('BrowserWindow module', function () {
     })
   })
 
+  describe('begin-sheet event', function (done) {
+    if (process.platform !== 'darwin') {
+      return
+    }
+    it('emits when window opens a sheet', function () {
+      let sheet
```

Comment:
This should be a variable declared at the root `describe` level and this window should be cleaned up in an `afterEach` to make sure it does not stick around.

```js
afterEach(function () {
  return closeWindow(sheet).then(function () { sheet = null })
})
```